vsphere/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,455 @@
+          See https://github.com/DataDog/integrations-core/blob/master/vsphere/datadog_checks/vsphere/metrics.py
+          for the list of collected metrics (do not prefix them with `vsphere`)
+          Note1: 'datastore' and 'cluster' are possible only with "collection_type: historical"
+          Note2: 'vm' and 'host' are possible only with "collection_type: realtime"
+          /!\ Use with parsimony, collecting per-instance metrics might be very expensive for big environments.
+        value:
+          default: none
+          example:
+            vm:
+              - <VM_REGEX>                 # Only possible with "collection_type: realtime"
+            host:
+              - <HOST_REGEX>               # Only possible with "collection_type: realtime"
+            datastore:
+              - <DATASTORE_REGEX>          # Only possible with "collection_type: historical"
+            cluster:
+              - <CLUSTER_REGEX>            # Only possible with "collection_type: historical"
+          type: object
+          properties:
+            - name: vm
+              type: array
+              items:
+                type: string
+            - name: host
+              type: array
+              items:
+                type: string
+            - name: datastore
+              type: array
+              items:
+                type: string
+            - name: cluster
+              type: array
+              items:
+                type: string            
+      - name: collect_tags
+        required: false
+        description: | 
+          If true, the vSphere Tags will be collected.
+          The final tag sent to Datadog is composed of the vSphere tags prefix + vSphere tag category as key and
+          vSphere tag name as value:
+            <TAG_PREFIX><TAG_CATEGORY>:<TAG_NAME>
+          Examples:
+          - os_type:windows
+          - application_name:my_app
+        value:
+          type: boolean
+          example: false
+      - name: tags_prefix
+        required: false
+        description: |
+          Custom tags attached to vSphere resources will be prefixed with this prefix when collected.
+          Example use cases:
+            - You have vSphere tags that might conflict with vSphere attributes or tags added by this integration or the
+              Datadog Agent
+            - You want a prefix to better recognise tags coming from vSphere
+          Examples with using `vtag_` prefix:
+          - vtag_os_type:windows            # where `os_type` is the category name and `windows` the tag name
+          - vtag_application_name:my_app    # where `application_name` is the category name and `my_app` the tag name
+          It's not recommend to use `vsphere_` as prefix, since it's already used by this integration
+          e.g. vsphere_host, vsphere_type, etc.
+        value:
+          type: string
+          default: "''"
+          example: ''
+      - name: collect_attributes 
+        required: false
```

Comment:
(nit) General comment, you can remove `required: false` since the default is already `false`.

Reply:
Seems there are still come `required: false`.

Reply:
I just removed these. Thanks!